app/src/main/java/com/academy/ramon/marvelcomicviewer/presenter/Presenter.java

```diff
@@ -0,0 +1,56 @@
+package com.academy.ramon.marvelcomicviewer.presenter;
+
+import com.academy.ramon.marvelcomicviewer.api.MarvelAPI;
+import com.academy.ramon.marvelcomicviewer.components.DaggerRetrofitComponent;
+import com.academy.ramon.marvelcomicviewer.components.RetrofitComponent;
+import com.academy.ramon.marvelcomicviewer.components.RetrofitModule;
+import com.academy.ramon.marvelcomicviewer.models.ComicResponse;
+import com.academy.ramon.marvelcomicviewer.models.ComicResults;
+import com.academy.ramon.marvelcomicviewer.models.Hero;
+import com.academy.ramon.marvelcomicviewer.models.HeroesResponse;
+import com.academy.ramon.marvelcomicviewer.util.ComicAdapter;
+import com.academy.ramon.marvelcomicviewer.util.HeroAdapter;
+
+import java.util.List;
+
+import rx.Observable;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+/**
+ * Created by Ramon on 1/3/2017.
+ */
+
+public class Presenter {
+
+    MarvelAPI service;
+    RetrofitComponent retrofitComponent = DaggerRetrofitComponent.builder().retrofitModule(new RetrofitModule()).build();
```

Comment:
@Mathreyu Avoid this kind of initializations and let dagger provide the service as a dependency as @jmsalcido suggested you.